src/components/breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -3,13 +3,25 @@ import { Svg } from '../svg/Svg';
 
 export interface IBreadcrumbLinkProps {
   name: string;
-  link: string;
+  link?: string;
+  onClick?: (props: IBreadcrumbLinkProps) => boolean;  // return false to cancel the href event
 }
 
 export class BreadcrumbLink extends React.Component<IBreadcrumbLinkProps, {}> {
+
+  private onClick(e: React.MouseEvent<HTMLAnchorElement>): boolean {
+    if (this.props.onClick && !this.props.onClick(this.props)) {
+      e.stopPropagation();
+      e.nativeEvent.stopImmediatePropagation();
+      e.preventDefault();
+      return false;
+    }
+    return true;
+  }
```

Comment:
I think this should be tested to improve coverage.

Reply:
Added